src/organisms/Address/Address.story.tsx

```diff
@@ -2,37 +2,22 @@ import { storiesOf } from '@storybook/react';
 import React, { Component } from 'react';
 
 import Address from './Address';
+import { Copyable } from 'src/molecules';
+import AddressLabelProps from 'src/atoms/EditableTitle/EditableTitle';
+import { Identicon } from 'src/atoms';
 
 const storyProps = {
   address: '0x4bbeEB066D09B7AEd07bF39EE20460DFa261520',
   title: 'John Doe',
-  truncate(children: string) {
-    return [
-      children.substring(0, 6),
-      '…',
-      children.substring(children.length - 4),
-    ].join('');
-  },
 };
 
-class AddressContainer extends Component {
-  public state = { title: storyProps.title };
-
-  public handleSubmit = (title: string) => this.setState({ title });
-
-  public render() {
-    const { title } = this.state;
-
-    return (
-      <Address {...storyProps} title={title} onSubmit={this.handleSubmit} />
-    );
-  }
-}
-
 storiesOf('Molecules', module).add('Address', () => (
   <>
-    <Address {...storyProps} />
-    <AddressContainer />
-    <Address title={storyProps.title} address="foo.eth" />
+    <Address
+      copyable={false}
+      editable={false}
+      address="myAddress"
+      truncate={true}
+    />
   </>
 ));
```

Comment:
It seems like only the icons are rendering in this story https://mycryptobuilds.com/ui/non-interactive-address-variant/?selectedKind=Molecules&selectedStory=Address&full=0&addons=1&stories=1&panelRight=0&addonPanel=%40storybook%2Faddon-a11y%2Fpanel